### PR TITLE
Minor clean-up in fe module

### PR DESCRIPTION
**Pull request by maxentile**
* Refactor some functions that hardcoded a specific way to compute works
(simplifying signatures from e.g. `EXP_loss(complex_du_dls, solvent_du_dls, lambda_schedule, true_dG, temperature=300 * unit.kelvin)` to `EXP_loss(complex_w_insert, solvent_w_insert, true_delta_f)`)
* Attic outdated / unused utilities
* Docstring typo fixes / readability tweaks
* Extract repeated code fragment for generating atom map colors + make deterministic

### CODE REVIEW

File: timemachine/fe/single_topology_v3.py [hunk 1]
```diff
@@ -577,15 +577,10 @@ def combine_masses(self):
...
-        Parameters
-        ----------
-        lamb: float
-            Lambda value
-
         Returns
         -------
-        np.array
-            shape of self.get_num_atoms()
+        masses: list
```
Comment (mcwitt) [inline timemachine/fe/single_topology_v3.py:582]: nit: `list of float`?

File: timemachine/fe/utils.py [hunk 2]
```diff
@@ -107,33 +107,36 @@ def draw_mol(mol, highlightAtoms, highlightColors):
     # display(SVG(svg))
 
 
-def plot_atom_mapping(mol_a, mol_b, core):
-    """from YTZ, Feb 1, 2021
+def get_atom_map_colors(core, seed=2022):
+    rng = np.random.default_rng(seed)
```
Comment (mcwitt) [inline timemachine/fe/utils.py:78]: 👍

File: timemachine/fe/utils.py [hunk 3]
```diff
@@ -107,33 +107,36 @@ def draw_mol(mol, highlightAtoms, highlightColors):
...
+    # TODO: replace random colors with colormap?
+    for (a_idx, b_idx), rgb in zip(core, rng.random((len(core), 3))):
         atom_colors_a[int(a_idx)] = tuple(rgb.tolist())
         atom_colors_b[int(b_idx)] = tuple(rgb.tolist())
 
+    return atom_colors_a, atom_colors_b
+
+
+def plot_atom_mapping(mol_a, mol_b, core, seed=2022):
+    """TODO: move this into a SingleTopology.visualize() or SingleTopology.debug() method?"""
```
Comment (mcwitt) [inline timemachine/fe/utils.py:91]: :+1:

**Review by mcwitt — APPROVED** [review]
Nice, this looks like a great set of changes / cleanups.

File: timemachine/fe/single_topology_v3.py (hunk 1 above)
Comment (maxentile) [reply timemachine/fe/single_topology_v3.py]: Addressed in https://github.com/proteneer/timemachine/pull/846/commits/026cde2d361c148b04c80ece1b6491a3ae605db8

**Pull request merged**